Libs/DICOM/Core/ctkDICOMDatabase.cpp

```diff
@@ -159,16 +159,6 @@ bool ctkDICOMDatabasePrivate::loggedExec(QSqlQuery& query, const QString& queryS
   {
     success = query.exec();
   }
-  if (!success)
-  {
-    QSqlError sqlError = query.lastError();
-    logger.debug("SQL failed\n Bad SQL: " + query.lastQuery());
```

Comment:
Errors should be still logged, probably at least at a warning level (unless there are a lot of false alarms, but I don't think there should be any).